### ISSUE TITLE
Arch on Surface Laptop Studio 2 doesn't power off 

### ISSUE THREAD

**Issue by anonemumei** [linux-surface/linux-surface#1507]
Hello Instead of leaving a comment only , I thought of opening an issue for this

[archlinux-log.txt](https://github.com/user-attachments/files/16403044/archlinux-log.txt)
[archlinux-log1.txt](https://github.com/user-attachments/files/16403045/archlinux-log1.txt)
I installed arch on surface laptop studio 2
I Installed kde DE and almost everything works fine, touchscreen, pen, sound etc.

However I have an issue with shutdown, the laptop doesn't power off. I waited for a while thinking there was some process taking place but no. The fan keeps working and I can hear the sound of the laptop still working.

By the way the laptop also didn't power off when I used Pop distro bootable USB. I needed Pop's gparted to shrink the C drive. So on both distros: Pop and archlinux the laptop doesn't power off.

I attached two logs file. Appreciate your support with this issue

[archlinux-log.txt](https://github.com/user-attachments/files/16403035/archlinux-log.txt)
[archlinux-log1.txt](https://github.com/user-attachments/files/16403036/archlinux-log1.txt)

**Comment by kwai-chang** [comment]
Same on my pro 9, but with solving this other issue, everything works fine

Blacklist the drivers, edit (or add) /etc/modprobe.d/blacklist.conf and add

blacklist intel-ipu6
blacklist intel-ipu6-isys

Booting is back, everything runs smooth
also with driver package mx-debian-firmware 24.08.10

**Comment by anonemumei** [comment]
> Same on my pro 9, but with solving this other issue, everything works fine
> 
> Blacklist the drivers, edit (or add) /etc/modprobe.d/blacklist.conf and add
> 
> blacklist intel-ipu6 blacklist intel-ipu6-isys
> 
> Booting is back, everything runs smooth also with driver package mx-debian-firmware 24.08.10

Thanks for your reply. Have been waiting!
Please show me detailed steps. I am kind of new to this.
Deeply appreciated

**Comment by anultravioletaurora** [comment]
Hey! I had this same issue on my SLS2 and [this user referenced a fix](https://github.com/linux-surface/linux-surface/issues/1272#issuecomment-2294501679) 😊

[Ubuntu's docs provide info](https://documentation.ubuntu.com/real-time/en/latest/how-to/modify-kernel-boot-parameters/) on how to update kernel parameters in GRUB, hope this helps!

**Comment by Anodyine** [comment]
yes I can confirm that adding pci=hpiosize=0 to the kernel parameters fixes this issue.
